WalletWasabi.Packager/MacSignTools.cs

```diff
@@ -0,0 +1,394 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace WalletWasabi.Packager
+{
+	public static class MacSignTools
+	{
+		public static void Sign()
+		{
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+			{
+				throw new NotSupportedException("This signing methon only valid on macOS!");
+			}
+
+			Console.WriteLine("Phase: finding the zip file on desktop which contains the compiled binaries from Windows.");
+
+			string desktopPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop);
+
+			var srcZipFileNamePattern = "Wasabi-osx-*.zip";
+			var files = Directory.GetFiles(desktopPath, srcZipFileNamePattern); //Example: Wasabi-osx-1.1.10.2.zip
+			if (files.Length != 1)
+			{
+				throw new InvalidDataException($"{srcZipFileNamePattern} file missing or there are more on Desktop! There must be exactly one!");
+			}
+			var zipPath = files[0];
+			var versionPrefix = zipPath.Split('-').Last().TrimEnd(".zip", StringComparison.InvariantCultureIgnoreCase); // Example: "/Users/user/Desktop/Wasabi-unsigned-1.1.10.2.dmg".
```

Comment:
The `TrimEnd` removes the `.zip` extension but the comment example says `.dmg`. I think the comment is not right.

Reply:
Fixed!